docs/index.md

```diff
@@ -6,69 +6,87 @@ This page is the top level entry point into the Substrate documentation.
+  Substrate makes it possible to supply custom consensus engines and also ships with several consensus mechanisms that
+  have been built on top of [Web3 Foundation research](https://w3f-research.readthedocs.io/en/latest/index.html).
+- **Remote procedure calls** allow blockchain users to interact with the network. Substrate provides HTTP and WebSocket
+  RPC servers.
+- **Telemetry** metrics are exposed by way of an embedded [Prometheus](https://prometheus.io/) server.
 
 ## Usage
 
 ![Technical Freedom vs Development Ease](assets/technical-freedom.png)
 
 Substrate is designed to be used in one of three ways:
 
-1. **With the Substrate Node**: You can run the pre-designed Substrate Node and configure it with a
-   genesis block that includes the default node runtime. In this case, you just need to configure a
-   JSON file and launch your own blockchain. This affords you the least amount of customization,
-   only allowing you to change the genesis parameters of the included runtime modules such as:
-   balances, staking, block-period, fees, governance, etc... For a tutorial on doing this, see
-   [Start a Private Network with Substrate](tutorials/start-a-private-network/index.md).
-
-2. **With the Substrate FRAME**: You can easily create your own custom blockchain using the FRAME. This affords
-   you a very large amount of freedom over your own blockchain's logic, letting you change
-   datatypes, select from the library of modules, and add your own custom modules. Much can be
-   changed without touching the block-authoring logic since it is directed through on-chain logic.
-   If this is the case, then the existing Substrate binary can be used for block authoring and
-   syncing. If the block authoring logic needs to be modified, then a new block-authoring binary
-   must be built as a separate project and used by validators. This is how the Polkadot relay chain
-   is built and should suffice for almost all needs in the near future. For a tutorial on this, see
+1. **With the Substrate Node**: You can run the pre-designed Substrate Node and configure it with a genesis block that
+   includes the default node runtime. In this case, you just need to configure a JSON file and launch your own
+   blockchain. This affords you the least amount of customization, only allowing you to change the genesis parameters of
+   the included runtime modules such as: balances, staking, block-period, fees, governance, etc... For a tutorial on
+   doing this, see [Start a Private Network with Substrate](tutorials/start-a-private-network/index.md).
+
+2. **With Substrate FRAME**: You can easily create your own custom runtime using FRAME. This affords you a very large
+   amount of freedom over your own blockchain's logic, letting you change data types, select from the library of
+   modules, and add your own custom modules. Much can be changed without touching the block-authoring logic since it is
```

Comment:
modules should potentially be `pallets` depending on how you are standardizing terminology in the docs.

I dont see much difference between the two personally

Reply:
technical is there any deviation in the terms/implementation or are they interchangeable? I ask for future style guides

Reply:
Well there is historical and literal reasons to use the term module.

Before we had "FRAME" and "Pallet", it was "SRML (Substrate Runtime Module Library)" and "Modules"

Crates and Modules are the official terms Rust uses "split code into logical units":
https://doc.rust-lang.org/1.0.0/book/crates-and-modules.html#basic-terminology:-crates-and-modules

So all Pallets are Rust modules/crates, but not all crates/modules are Pallets.

Also a pallet may itself have modules like the benchmarking, test, or mock module which are imported with:

```
mod test;
mod benchmarking;
mod mock;
```

etc...


So its all about specificity and whatever you think is best.

Reply:
Probably best to go with "pallet" here on second review.

Reply:
I agree that pallet is better here.